modules/pipeline/services/pipelinesvc/create_v2.go

```diff
@@ -46,7 +46,7 @@ func (s *PipelineSvc) CreateV2(req *apistructs.PipelineCreateRequestV2) (*spec.P
 		return nil, err
 	}
 
-	if err := s.CreatePipelineGraph(p); err != nil {
+	if err = s.CreatePipelineGraph(p); err != nil {
```

Comment:
why replace `:=` with `=`

Reply:
Because the error has been declared before, no need to redeclare